packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -797,8 +796,9 @@ mixin WidgetInspectorService {
   void registerServiceExtension({
```

Comment:
make this method protected as it doesn't make sense to call outside of this class or subclasses.